code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm

```diff
@@ -1,7 +1,8 @@
 #define CHALLENGE_TELECRYSTALS 280
```

Comment:
This still gives starting 280 telecrystals regardless of the pop, 20 crew would be horrably out classed by that + the 60 default nukies get

Reply:
It's an inverse calculation, Trilby. That's the max possible telecrystals, and it decreases from there.

Reply:
see lines 70 to 74, more or less what ramsay said.